src/core.rs

```diff
@@ -369,6 +369,10 @@ impl Core {
                 //     true
                 // });
                 trace!(" -----------------------------------");
+
+                trace!(" -----------------------------------");
+                trace!("| Number of proxy nodes : {}", self.proxy_map.len());
+                trace!(" -----------------------------------");
```

Comment:
It was more for ensuring it goes back to zero when we have group size routing nodes in the table, but fair point.